javascript/packages/orchestrator/src/chainSpec.ts

```diff
@@ -119,6 +132,8 @@ export async function addBalances(specPath: string, nodes: Node[]) {
       }
     }
 
+    runtimeConfig.balances.balances = Object.entries(balanceMap);
```

Comment:
Why do we have ```balances.balances``` here ?

Reply:
This is how the chainspec is specified in [polkadot](https://github.com/paritytech/polkadot/blob/f963646940e3c38e359d1ecf76478d1fed36021c/node/service/src/chain_spec.rs#L324-L325) as far as I know